erpnext/accounts/report/accounts_receivable/accounts_receivable.py

```diff
@@ -267,6 +267,18 @@ def update_voucher_balance(self, ple):
 				row.invoiced_in_account_currency += amount_in_account_currency
 		else:
 			if self.is_invoice(ple):
+				# when invoice has is_return marked
+				if self.invoice_details.get(row.voucher_no, {}).get("is_return"):
+					# for Credit Note
+					if row.voucher_type == "Sales Invoice":
+						row.credit_note -= amount
+						row.credit_note_in_account_currency -= amount_in_account_currency
+					# for Debit Note
+					else:
+						row.invoiced -= amount
```

Comment:
amount might already negative, subtracting it would in effect add up the value(?)